modules/ingestor/src/graph/sbom/spdx.rs

```diff
@@ -157,7 +159,9 @@ impl SbomContext {
                     "cpe22Type" => match Cpe::from_str(&r.reference_locator) {
                         Ok(cpe) => {
                             refs.push(PackageReference::Cpe(cpe.uuid()));
-                            cpes.add(cpe);
+                            cpes.add(cpe.clone());
+                            // assign only the first cpe for now
```

Comment:
Can you expand a little bit on rationale ... is there a TODO here ?

Reply:
Add some more context. Thanks